server/src/profile/importer.rs

```diff
@@ -0,0 +1,146 @@
+use crate::{embedded_oidc, endpoints, sample_data};
+use actix_web::{
+    body::MessageBody,
+    dev::{ConnectionInfo, Url},
+    error::UrlGenerationError,
+    get, middleware, web,
+    web::Json,
+    HttpRequest, HttpResponse, Responder, Result,
+};
+use anyhow::Context;
+use bytesize::ByteSize;
+use futures::{FutureExt, StreamExt};
+use std::{
+    fmt::Display, fs::create_dir_all, path::PathBuf, process::ExitCode, sync::Arc, time::Duration,
+};
+use tokio::task::JoinHandle;
+use trustify_auth::{
+    auth::AuthConfigArguments,
+    authenticator::Authenticator,
+    authorizer::Authorizer,
+    devmode::{FRONTEND_CLIENT_ID, ISSUER_URL, PUBLIC_CLIENT_IDS},
+    swagger_ui::{swagger_ui_with_auth, SwaggerUiOidc, SwaggerUiOidcConfig},
+};
+use trustify_common::{config::Database, db, model::BinaryByteSize};
+use trustify_infrastructure::{
+    app::{
+        http::{HttpServerBuilder, HttpServerConfig},
+        new_auth,
+    },
+    endpoint::Trustify,
+    health::checks::{Local, Probe},
+    tracing::Tracing,
+    Infrastructure, InfrastructureConfig, InitContext, Metrics,
+};
+use trustify_module_graphql::RootQuery;
+use trustify_module_importer::server::importer;
+use trustify_module_ingestor::graph::Graph;
+use trustify_module_storage::{
+    config::{StorageConfig, StorageStrategy},
+    service::{dispatch::DispatchBackend, fs::FileSystemBackend, s3::S3Backend},
+};
+use trustify_module_ui::{endpoints::UiResources, UI};
+use utoipa::openapi::{Info, License};
+use utoipa::OpenApi;
+use utoipa_rapidoc::RapiDoc;
+use utoipa_redoc::{Redoc, Servable};
+
+/// Run the API server
+#[derive(clap::Args, Debug)]
+pub struct Run {
```

Comment:
To me, it's confusing to have both this module *and* the server's `with_importer` argument. I would think one or the other would suffice.

My vote would be for keeping just the `with_importer` feature of the server and delete this module. To run a dedicated importer, we'd simply set the `with_importer` to true on one of the deployed `api` servers.